docs/code/ExplainabilityCookBook.md

```diff
@@ -0,0 +1,90 @@
+# Explainability CookBook
+
+This document serves to provide different ways to use the explainability features of ML.NET.
+
+All of these samples will use the [housing data](https://github.com/dotnet/machinelearning/blob/master/test/data/housing.txt) and will reference the below data schema class and pipeline.
+
+```csharp
+public class HousingData
+{
+    public float MedianHomeValue { get; set; }
+    public float CrimesPerCapita { get; set; }
+    public float PercentResidental { get; set; }
+    public float PercentNonRetail { get; set; }
+    public float CharlesRiver { get; set; }
+    public float NitricOxides { get; set; }
+    public float RoomsPerDwelling { get; set; }
+    public float PercentPre40s { get; set; }
+    public float EmploymentDistance { get; set; }
+    public float HighwayDistance { get; set; }
+    public float TaxRate { get; set; }
+    public float TeacherRatio { get; set; }
+}
+
+MLContext context = new MLContext();
+
+IDataView data = context.Data.LoadFromTextFile("./housing.txt", new[]
```

Comment:
So this code for loading and specifying the data class would be necessary in the .cs file to run your tests. However, I think we should not add it to the .md file. We have other sections of the cookbook and samples in which we illustrate that.